test/Sentry.Tests/Internals/MainSentryEventProcessorTests.cs

```diff
@@ -63,6 +63,43 @@ public void Process_SendDefaultPiiTrueIdEnvironmentTrue_UserNameSet()
             Assert.Equal(Environment.UserName, evt.User.Username);
         }
 
+        [Fact]
+        public void Process_SendDefaultPiiTrueIdEnvironmentTrueUserIpNull_UserIpSetDefaultIp()
+        {
+            //Arrange
+            var evt = new SentryEvent();
+
+            _fixture.SentryOptions.SendDefaultPii = true;
+            _fixture.SentryOptions.IsEnvironmentUser = true;
+            var sut = _fixture.GetSut();
+
+            //Act
+            _ = sut.Process(evt);
+
+            //Assert
+            Assert.Equal(sut.UserIpDefault, evt.User.IpAddress);
+        }
+
+        [Fact]
+        public void Process_SendDefaultPiiTrueIdEnvironmentTrueUserIpSet_UserIpIgnoreDefaultIp()
```

Comment:
```suggestion
        public void Process_SendDefaultPiiTrueAndUserIpSet_UserIpIgnoreServerInferredIp()
```

Reply:
Just wondering, 
https://github.com/getsentry/sentry-dotnet/blob/4c6fdac57a5cd8179d0af6fb8e219c56767a2c90/test/Sentry.Tests/Internals/MainSentryEventProcessorTests.cs#L104

`Process_SendDefaultPiiTrueIdEnvironmentFalse_UserNameNotSet
`
Should become 
`Process_SendDefaultPiiTrueIdAndEnvironmentFalse_UserNameNotSet `
?